packages/create-sitecore-jss/src/templates/nextjs-personalize/.env.production

```diff
@@ -0,0 +1,5 @@
+# Timeout for CDP requests to respond within (ms)
```

Comment:
Do we need the .env.production file? We already have a fallback of 250 in the personalze.ts file and in the SDK middleware itself. We can leave addl production config up to their deployment platform of choice (e.g. set in Vercel project).

Reply:
Yes - if we want it to be configurable. If variables are put into regular env file that would be the value used over others. So I've put both dev and prod env files to differentiate between usage scenarios.

Reply:
Hmm... I guess I'm just leery of introducing more of these .env files which start dictating a customer's setup for prod. 

Maybe it's best we just go back to how you had it originally in our single .env file and add a comment that these should be tuned down for production? This falls more in line with other variables which need to be adjusted per environment (e.g. PUBLIC_URL).